package.json

```diff
@@ -26,6 +26,8 @@
     "axios": "^0.19.0",
     "babel-plugin-transform-define": "^1.3.1",
     "color": "^3.0.0",
+    "eslint": "^5.16.0",
+    "eslint-plugin-react": "^7.13.0",
```

Comment:
are they used somewhere now? what about lodash - we removed it below, should we remove it from the package.json?

Reply:
we installed it for ourselves, we want to set up a beautiful code for your code style

Reply:
it should be done as a separate discussion, separate PR. let's remove it from this PR for now.

Reply:
Ok